src/common/loki.h

```diff
@@ -30,10 +30,14 @@
 #define LOKI_H
 
 #include <string>
+#include "version.h"
 
 #define LOKI_HOUR(val) ((val) * LOKI_MINUTES(60))
 #define LOKI_MINUTES(val) val * 60
 
+#define LOKI_VERSION_INTEGER(major, minor, patch) ((major * 1000) + (minor * 100) + patch)
+#define LOKI_DEPRECATE_AT_VERSION(major, minor, patch) (LOKI_VERSION_INTEGER(major, minor, patch) >= LOKI_VERSION_INTEGER(DEF_LOKI_VERSION_MAJOR, DEF_LOKI_VERSION_MINOR, DEF_LOKI_VERSION_PATCH))
```

Comment:
This macro is bringing a header define version dependency back in, after a fair bit of effort went into getting header define versions *out* of the code in favour of run-time values.  It also seems unnecessary -- I see no reason (see later comments) why all of the things it is deprecating can't just be removed right away without breaking anything.

Reply:
Also: we deprecate things at hard forks, not versions.

Reply:
Oh ok. X.1.X would be a good time to deprecate as people would upgrade and have time to fix their RPC scripts. 

Thought doing it was a novel way to deprecate stuff. Undoing.